templates/CRM/Admin/Form/Extensions.tpl

```diff
@@ -22,14 +23,26 @@
         {ts}Installing this extension will provide you with new functionality.{/ts} {ts}Do you want to continue?{/ts}
       </div>
    {/if}
-   {if $action eq 2}
-     <div class="messages status no-popup">
+   {if $action eq 8}
+      <div class="messages status no-popup">
         {icon icon="fa-info-circle"}{/icon}
-        {ts}Downloading this extension will provide you with new functionality. Please make sure that the extension you're installing or upgrading comes from a trusted source.{/ts} {ts}Do you want to continue?{/ts}
+        {ts}WARNING: Uninstalling this extension might result in the loss of all records which use the extension.{/ts} {ts}This may mean the loss of a substantial amount of data, and the action cannot be undone. Please review the extension information below before you make final decision.{/ts} {ts}Do you want to continue?{/ts}
```

Comment:
I know it predates this PR but "before you make final decision" sounds like broken English.

It doesn't even add much to the conversation, maybe just cut it.

Reply:
@christianwach sure, you can cut the whole sentence.
WARNING: Cutting the whole sentence will cut the whole sentence. Read above sentence before cutting.